### ISSUE TITLE
Fix "hide page title" option and hero section on preview mode

### ISSUE THREAD

**Issue by zainabD** [ASU-KE/UDS-WordPress-Theme#135]
Page title will still show on preview mode even if "hide page title" option was selected. 
Also the hero image section will not show on preview mode.

**Comment by zainabD** [comment]
The issue only occurs with hero section and hide page title option.
The Hero image and hide page title option doesn't always work on the preview mode.
- preview mode should show you what the page would look like before you update and save the changes. 
- If I create a new page, and add a new hero image, or hide the page title, everything works fine on preview mode (before publishing the page)
- If the page was published, and before updating or adding anything, if I go to preview mode, the hero image and page title will not work, if I go to preview mode again, everything works. If I go to preview mode a third time, nothing works .. and so on.
- If the page was published, and have a hero image, if I add anything to the page content, Before I hit update, I go to preview mode, then I can see the new updates but not the hero image. the same thing happens if I keep going back to preview mode as mentioned in the note above. even if I hit update to save the new updates, hero image will not work on the first preview, but works on the second try, doesn't work on the third and so on.

**Comment by zainabD** [comment]
Everything works fine with a new page (before publishing): 

https://user-images.githubusercontent.com/15880606/112073239-e9ac3400-8b30-11eb-917e-0c12da0afa9a.mov

**Comment by zainabD** [comment]
If the page was published and we are adding content or updating the page:

https://user-images.githubusercontent.com/15880606/112073302-0c3e4d00-8b31-11eb-8d93-f6bc5bf01c86.mov

**Comment by wmcconnell** [comment]
I'm closing this issue for the time being as part of cleaning up the queue. Half of this issue has been solved by the use of the Hero block, and we can move the other half into a separate issue. I'll put this in my notes and submit a new issue once the cleanup is done.
